buildprocess.tex

```diff
@@ -4,6 +4,73 @@
 
 \indextext{build|(}%
 
+\rSec1[build.import]{Import relationship between translation units}
+
+\pnum The name of a module used in the ``import'' and ``module''
+keywords is the ``logical name'' of a module.
+
+\pnum When a translation unit imports a named module, the translation
+unit doing the import has an import dependency on the translation unit
+providing the importable unit identified via the ``logical name''.
+
+\pnum The translation of an importable unit is externalized into an
+intermediate artifact, the Built Module Interface file (BMI). That
+file is then consumed when importing a named module.
+
+\pnum BMI files can be constructed in a way to optimize for the
+efficiency of the reuse in the import operation, which may result in a
+limitation on the number of scenarios where that file can be used.
+
+\rSec1[build.arguments]{Compiler arguments and the import relationship}
+
+\pnum ``ABI coherency'' is the expected level of coherency in the
+compiler arguments for the translation of the code in the purview of a
+module and all translation units importing that module. When that
+level of coherency is not met, the final program may fail to link or
+it may link and have undefined behavior at runtime.
+
+\pnum ``BMI coherency'' is the expected level of coherency in the
+compiler arguments for a BMI file to be used when importing a
+module. When that level of coherency is not met, the translation will
+fail because the BMI cannot be loaded.
+
```

Comment:
Suggestion: Examples where ABI and BMI coherency are and are not met would be helpful.

Reply:
Created https://github.com/cplusplus/modules-ecosystem-tr/issues/25 to follow up on this